packages/client/hmi-client/src/services/concept.ts

```diff
@@ -51,6 +52,21 @@ async function getCuriesEntities(curies: Array<string>): Promise<Array<DKG> | nu
 	}
 }
 
+async function searchCuriesEntities(query: string): Promise<Array<DKG> | null> {
+	try {
+		const response = await API.get('/mira/search', {
+			params: {
+				q: query
+			}
+		});
+		if (response?.status !== 200) return null;
+		return response?.data ?? null;
+	} catch (error) {
+		logger.error(error, { showToast: false });
+		return null;
```

Comment:
Why not returning an empty Array? This would simplify a lot of test down the line.

Reply:
changed